src/components/atoms/bar-chart-content.tsx

```diff
@@ -102,6 +104,28 @@ export const BarChartContent: FC<BarChartContentProps> = ({
     <TrendLine lineWidth={3} color={primaryColor} {...props} />
   );
 
+  const Label: FC<{
+    data: any;
+    y: (value: number) => void;
+    x: () => void;
+    bandwidth: number;
+  }> = (props) => {
+    const {x, y, bandwidth, data} = props;
+    return data.map((value: {value: number}, index: number) =>
+      index === data.length - 1 ? (
+        <View
+          accessible={true}
+          style={[styles.label, {top: y(value.value) - 35}]}>
```

Comment:
If you make the `right` here relative to `bandwidth / 2` it should line up perfectly on all size devices and if the amount of data or offset changes